components/dashboard/src/components/ContextMenu.tsx

```diff
@@ -50,12 +51,16 @@ function ContextMenu(props: ContextMenuProps) {
                 {props.children}
             </div>
             {expanded?
-                <div className={`z-50 ${props.width || 'w-40'} bg-white absolute py-2 right-0 flex flex-col border border-gray-200 rounded-lg space-y-2`}>
```

Comment:
**thought**: In the end, this dropdown should use the same spacing as the workspace more actions dropdown as described in https://github.com/gitpod-io/gitpod/pull/3496#pullrequestreview-616162423.

> Let's change the width of the dropdown of the more actions to `w-48`. This will also use the same width as the side menu. Also, let's change the dropdown spacing from `space-y-2` to `space-y-1` and `py-2` to `py-1`.

Reply:
I went away from the inbetween spacing, because separators would be hard to align in the middle. They are no based on borders.